<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewAccount</class>
 <widget class="QDialog" name="CreateNewAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Create New Account</string>
   </property>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Phone Number (no dashes)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Email Address</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Provider</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verizon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T-Mobile</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Metro PCS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sprint</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>firstName</tabstop>
  <tabstop>lastName</tabstop>
  <tabstop>phone</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>email</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>username</sender>
   <signal>editingFinished()</signal>
   <receiver>pushButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>password</sender>
   <signal>editingFinished()</signal>
   <receiver>pushButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>firstName</sender>
   <signal>editingFinished()</signal>
   <receiver>pushButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastName</sender>
   <signal>editingFinished()</signal>
   <receiver>pushButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phone</sender>
   <signal>selectionChanged()</signal>
   <receiver>pushButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>pushButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>email</sender>
   <signal>editingFinished()</signal>
   <receiver>pushButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
